### ISSUE TITLE
Replace "react-diff-viewer" dependency with "react-diff-viewer-continued"

### ISSUE THREAD

**Issue by tadwohlrapp** [boazpoolman/strapi-plugin-config-sync#75]
### Describe the bug

I get the following warnings when installing strapi-plugin-config-sync:

```
warning "strapi-plugin-config-sync > react-diff-viewer@3.1.1" has incorrect peer dependency "react@^15.3.0 || ^16.0.0".
warning "strapi-plugin-config-sync > react-diff-viewer@3.1.1" has incorrect peer dependency "react-dom@^15.3.0 || ^16.0.0".
```

[`react-diff-viewer`](https://github.com/praneshr/react-diff-viewer/issues/166) seems to be abandoned (last update was in May 2020) and there is a successor fork called [`react-diff-viewer-continued`](https://github.com/Aeolun/react-diff-viewer-continued) ([npm package](https://www.npmjs.com/package/react-diff-viewer-continued)).

### Steps to reproduce the behavior

1. Use yarn
2. Have a React version higher than v16.x installed (or no React installed, which would cause a `has unmet peer dependency` warning)
3. Run `yarn add strapi-plugin-config-sync`

### Expected behavior

Use currently maintained dependencies which don't cause warnings

### Screenshots

![image](https://user-images.githubusercontent.com/2788192/200546046-97a1954e-27c7-4965-9807-dbbb907f5242.png)

### System

- Node.js version: 16.18.0 LTS
- NPM version: 8.19.2
- Strapi version: 4.4.7
- Plugin version: 1.0.4
- Database: SQLite
- Operating system: macOS

**Comment by boazpoolman** [comment]
Excellent suggestion. It makes sense to switch to the new package.

**Comment by boazpoolman** [comment]
Made the change directly on the master branch. [See commit](https://github.com/boazpoolman/strapi-plugin-config-sync/commit/ad7fc93b8459143c43b8f02e8fe67ee75003ef83).
I'll release it with the next release of the plugin :)

**Comment by boazpoolman** [comment]
This was released with [v1.1.0](https://github.com/boazpoolman/strapi-plugin-config-sync/releases/tag/1.1.0) of the plugin.
